applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -40,7 +40,7 @@ static struct bt_audio_unicast_group *unicast_group;
 static struct bt_codec *remote_codecs[CONFIG_BT_AUDIO_UNICAST_CLIENT_PAC_COUNT];
 static struct bt_audio_ep *sinks[CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SNK_COUNT];
 static struct net_buf_pool *iso_tx_pools[] = { LISTIFY(CONFIG_BT_ISO_MAX_CHAN,
-								 NET_BUF_POOL_PTR_ITERATE, (,)) };
+						       NET_BUF_POOL_PTR_ITERATE, (, )) };
```

Comment:
Hmmm, it took me a little time to make this pass the checkpatch.pl

Reply:
Now this passes clang-format, but now checkpatch. We must fix this before merging into NCS